addons/web/static/src/views/fields/float/float_field.js

```diff
@@ -59,6 +60,9 @@ export class FloatField extends Component {
         ) {
             return this.value;
         }
+        if (this.value instanceof FieldOperator) {
```

Comment:
With what I have in mind, this shouldn't be necessary in Fields, as the value would always be a number here.